specification/resources/apps/models/app_propose_response.yml

```
@@ -0,0 +1,46 @@
+type: object
+
+properties:
+  app_is_static:
+    type: boolean
+    description: Indicates whether the app is a static app.
+    example: true
+
+  app_name_available:
+    type: boolean
+    description: Indicates whether the app name is available.
+    example: true
+
+  app_name_suggestion:
+    type: string
+    description: The suggested name if the proposed app name is unavailable.
+    example: newName
+
+  existing_static_apps:
+    type: string
+    description: The maximum number of free static apps the account can have.
+      We will charge you for any additional static apps.
+    example: 2
```

```suggestion
    example: '2'
```
Since the `type` is `string`, this should be quoted.